datastore/load_test.go

```diff
@@ -584,6 +569,36 @@ func TestLoadToInterface(t *testing.T) {
 	}
 }
 
+func TestTimezone(t *testing.T) {
+	// Expect Local times to be represented in UTC
+	src := &pb.Entity{
+		Key: keyToProto(testKey0),
+		Properties: map[string]*pb.Value{
+			"Time": {ValueType: &pb.Value_TimestampValue{TimestampValue: &timestamppb.Timestamp{Seconds: 1605504600}}},
+		},
+	}
+
+	dst := &struct{ Time time.Time }{
+		Time: time.Time{},
+	}
+	want := &struct{ Time time.Time }{
+		Time: time.Unix(1605504600, 0).In(time.UTC),
+	}
+
+	err := loadEntityProto(dst, src)
+
+	if err != nil {
+		t.Fatalf("loadEntityProto: %v", err)
+	}
+
+	gotZone, _ := dst.Time.Zone()
```

Comment:
Maybe actually compare dst to want as well, and note why you need to compare the time zones separately.